back/src/answer/answer.module.ts

```diff
@@ -0,0 +1,12 @@
+import { Module, forwardRef } from '@nestjs/common';
+import { AnswerService } from './answer.service';
+import AnswerResolver from './answer.resolver';
+import { UserModule } from 'src/user/user.module';
+import { PrismaModule } from 'src/prisma/prisma.module';
+
+@Module({
+  imports: [forwardRef(() => UserModule), PrismaModule],
```

Comment:
Why did you use forwardRed?

Reply:
I removed it, I had at some time a circular dependency, ty!